frontend/src/pages/modelCatalog/screens/ModelDetailsPage.tsx

```diff
@@ -46,10 +61,33 @@ const ModelDetailsPage: React.FC = conditionalArea(
         </Breadcrumb>
       }
       title={
-        <TitleWithIcon
-          title={decodedParams.modelName || ''}
-          objectType={ProjectObjectType.registeredModels}
-        />
+        <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+          <FlexItem>
+            <BrandImage src={model?.logo ?? ''} alt="" />
+          </FlexItem>
+          <FlexItem>
+            <Stack>
+              <StackItem>
+                <Flex
+                  spaceItems={{ default: 'spaceItemsSm' }}
+                  alignItems={{ default: 'alignItemsCenter' }}
+                >
+                  <FlexItem>{decodedParams.modelName}</FlexItem>
+                  {model && (
+                    <Label variant="outline" icon={<TagIcon />}>
+                      {getTagFromModel(model)}
+                    </Label>
+                  )}
+                </Flex>
+              </StackItem>
+              {model && (
+                <StackItem>
+                  <Content component={ContentVariants.small}>Provided by {model.provider}</Content>
```

Comment:
Hmm. I would have expected this to use the `source` property of the ModelCatalogSource (currently "Red Hat" in our data), but I didn't realize we also had the model's `provider` property (currently "IBM" in our data). According to the mocks I think we do want these granite models to be "provided by Red Hat", but it probably makes sense to keep this behavior as `model.provider` and change the ConfigMap to say Red Hat instead of IBM. That way we have the flexibility for different "providers" inside one ModelCatalogSource, which is what we'll want when we introduce the "Third Party" source with a Neural Magic provider.

@pboyd do you think it makes sense to change our manifest configmap to have these models say `provider: Red Hat` instead of IBM?